.github/workflows/check-outdated-content.yml

```diff
@@ -0,0 +1,159 @@
+# Contribution by Cloud Native Guatemala folks :)
+# This workflow will check if a localized content is outdated or not
+# by comparing English content in the old branch and the latest branch.
+---
+name: Check outdated content
+
+on:
+  pull_request:
+
+
+jobs:
+  check-outdated-content:
+    name: Check outdated content
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0 # fetch all history for all tags and branches
+    - name: Detecting pending translations
+      shell: bash
+      run: |
+        ##### DEBUG section, this will be removed later ###########
+        ls -al
+        git status
+        git branch
+        
+        # Default environment variables 
+        echo "GITHUB_REF: $GITHUB_REF"
+        echo "Extract branch: ${GITHUB_REF#refs/}"
+        
+        # `github` context information
+        echo "(DEBUG) github.ref: ${{github.ref}}"
+        echo "(DEBUG) github.head_ref: ${{github.head_ref}}"
+        echo "(DEBUG) github.base_ref: ${{github.base_ref}}"
+        #####################################################
+
+        OUTPUT_DIR="./outdated"
+        CONTENT_DIR="website/content"
+        languages=("es" "zh")
+        base_lang="en"
+        cd $CONTENT_DIR
+
+        echo "(DEBUG) Current folder: "$(pwd)
+
+        for lang in ${languages[@]}; do
+            touch $lang.txt
+        done
```

Comment:
Do you mind explaining this code?  We create a txt file per translation language. So we end up with zh.txt, es.txt ...  why do we need these files? Probably we add this info to the issue descriptions, right.

Reply:
Sure we have to document that part.

Reply:
To get things going I think that it's a non-critical issue and can be an improvement that can be added in a subsequent PR. The work will be tracked in #430  , therefore resolving this thread.